Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/result/AggregatedRiskPerDateResult.kt

```diff
@@ -0,0 +1,10 @@
+package de.rki.coronawarnapp.risk.result
+
+import de.rki.coronawarnapp.server.protocols.internal.v2.RiskCalculationParametersOuterClass
+
+data class AggregatedRiskPerDateResult(
+    val dateMillisSinceEpoch: Long,
```

Comment:
Can we use some date-like data type here such as `Instant` (like you used in `AggregateRiskResult`)?

Reply:
Never mind, I saw your other comment above that it's easy to to calculations if it's `Long`. As long as the calculation does not expose `Long` in the end, I think it's fine (looks like it's `Instant` 👍 )